src/uniprotkb/components/protein-data-views/FreeTextView.tsx

```diff
@@ -20,50 +21,82 @@ const needsNewLineRE = /^\).\s+/;
 
 const getEntryPathForCitation = getEntryPathFor(Namespace.citations);
 
+export const ACCommentView = ({ string }: { string: string }) => {
+  // This replaces any occurrences of "AC <accession>" with "AC "<link to the accession>
+  const acRE = new RegExp(`(AC ${uniProtKBAccessionRegEx.source})`, 'i');
```

Comment:
```suggestion
const acRE = new RegExp(`(AC ${uniProtKBAccessionRegEx.source})`, 'i');

export const ACCommentView = ({ string }: { string: string }) => {
  // This replaces any occurrences of "AC <accession>" with "AC "<link to the accession>
```

Reply:
I think this is what @dlrice was referring to here https://github.com/ebi-uniprot/uniprot-website/pull/506#discussion_r767761224
I tried it locally and had the same issue: I think webpack is processing the dependency in an order that means that `uniProtKBAccessionRegEx` isn't initialised at the time... played with it for a bit locally but couldn't find a solution either.

Reply:
Ah, it's a circular dependency as we've seen before some times. Maybe these regular expressions could be defined directly in the utils module then?

Reply:
Exported from utils module now